platform.go

```diff
@@ -174,6 +206,10 @@ func NewBindings(platformDir string) (Bindings, error) {
 		return NewBindingsFromPath(path)
 	}
 
+	if content, ok := os.LookupEnv(EnvVcapServices); ok {
```

Comment:
1. Any objections to having this happen last? (or first). I'm not sure I like it happening in the middle of looking for service bindings. Or was there a specific reason you put this in the middle?

2. Please update the comment to reflect how things are loaded.